packages/odyssey-react/src/components/Heading/Heading.theme.ts

```diff
@@ -13,7 +13,7 @@
 import type { ThemeReducer } from "@okta/odyssey-react-theme";
 
 export const theme: ThemeReducer = (theme) => ({
-  MarginBlockEnd: theme.SpaceEmXs,
+  MarginBlockEnd: theme.Space2,
```

Comment:
Yup, the values in Figma are now proportional to the rem base rather than being proportional to their element, so I've made the same changes throughout.